src/main/java/org/javarosa/core/model/SelectChoice.java

```diff
@@ -91,15 +111,33 @@ public String getValue() {
     }
 
     public String getChild(String childName) {
-        if (item != null) {
-            TreeElement child = item.getChild(childName, 0);
-            if (child != null) {
-                return child.getValue().getDisplayText();
-            }
+        if (item == null) {
+            throw new IllegalStateException("Can only get the child of a choice from a select from itemset");
         }
+
+        TreeElement child = item.getChild(childName, 0);
+        if (child != null) {
+            return child.getValue().getDisplayText();
+        }
+
         return null;
     }
 
+    public Map<String, String> getAdditionalChildren() {
+        if (item == null) {
+            throw new IllegalStateException("Can only get the child of a choice from a select from itemset");
+        }
+
+        Map<String, String> children = new LinkedHashMap<>();
+        for (int i = 0; i < item.getNumChildren(); i++) {
+            TreeElement child = item.getChildAt(i);
+            if (!child.getRef().getNameLast().equals(labelRefName)) {
```

Comment:
This *only* filters out whatever the `label` ref is. I briefly considered filtering out `geometry` but that's static so it feels like Collect should do it. I initially was going to filter out the `value` as well but actually it can be helpful to see, I think.